owner/src/main/java/org/the_chance/honeymart/ui/products/composables/ProductCard.kt

```diff
@@ -27,6 +25,7 @@ fun ProductCard(
     imageUrl: String,
     productName: String,
     productPrice: String,
+    numberOfItems: String,
```

Comment:
Do not add this part because this already has been deleted from the design and backend. make sure to not make any changes in composable related to other screens.